pyproject.toml

```diff
@@ -17,3 +54,6 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 version_scheme = "post-release"
 local_scheme = "node-and-date"
+
+[tool.setuptools.packages.find]
```

Comment:
Configuring setuptools in pyproject.toml is still in beta, but it seems to work just fine

Reply:
Conveniently, `include_package_data` is true by default in pyproject.toml.

Reply:
Finding namespace packages is also turned on by default in pyproject.toml. https://setuptools.pypa.io/en/latest/userguide/datafiles.html?highlight=find_namespace_packages#subdirectory-for-data-files